src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -7,7 +7,7 @@ context of the academic literature:
 > Pernet, C. R., Appelhoff, S., Gorgolewski, K.J., Flandin, G., Phillips, C.,
 > Delorme, A., Oostenveld, R. (2019). **EEG-BIDS, an extension to the brain
 > imaging data structure for electroencephalography**. Scientific data, 6.
-> doi: [10.1038/s41597-019-0104-8](https://doi.org/10.1038/s41597-019-0104-8)
+> [doi:10.1038/s41597-019-0104-8](https://doi.org/10.1038/s41597-019-0104-8)
```

Comment:
can you please revert this change, so that it does not create a conflict for #630 ?